battery.h

```diff
@@ -12,13 +13,17 @@ class Battery
     void increase_battery(int);
     int get_battery();
     int get_range();
+    void CheckRange();
+    void on();
```

Comment:
Have it pass in the state so we know which warning it is referring to. We might move the Battery Warning into this class as well and remove the entirety of the warning class, and make a motor class. This way when we receive signals we know where to route the information and we aren't sending the same information to 2 places. 
```suggestion
    void on(warnings warning);
```